src/track/beatmap.cpp

```diff
@@ -631,7 +631,7 @@ void BeatMap::scaleQuadruple() {
         // Need to not accrue fractional frames.
         int distance = it->frame_position() - prevBeat.frame_position();
         Beat beat;
-        for (i=1; i<=3, i++;) {
+        for (int i=1; i<=3, i++;) {
```

Comment:
Good point -- it would be great if you submit a PR to update the manual too!
https://github.com/mixxxdj/manual